include/aws/common/thread.h

```diff
@@ -15,11 +15,12 @@
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
-
 #include <aws/common/common.h>
 
 #ifndef _WIN32
 #    include <pthread.h>
+#else
+#    include <windows.h>
```

Comment:
NONONONONONO

It should be pretty easy to F12 on INIT_ONCE and see what it actually is. It's probably a void *.

Reply:
👍